### PR TITLE
Added Jaccard coefficient metric for multiclass target in segmentation

### PR DESCRIPTION
**Pull request by Hazem-Ahmed-Abdelraouf**

### CODE REVIEW

**Comment by jph00** [comment]
Thank you! Can you please place your new class in a separate cell, and add tests for it underneath?

**Comment by Hazem-Ahmed-Abdelraouf** [comment]
> Thank you! Can you please place your new class in a separate cell, and add tests for it underneath?

You are most welcome, done Alhamdulillah (all praises are due to Allah) and ready for your review.

**Review by jph00 — CHANGES_REQUESTED** [review]
Thanks for the tests. You don't need the `clone` calls. You've accidentally deleted the cell outputs of some other cells. We need those for the documentation.

**Comment by Hazem-Ahmed-Abdelraouf** [comment]
Sorry for that, I reverted these changes locally, but before I push I wanted to ask about the `clone` calls. Do you mean I change it from this:
```
x1a = torch.ones(20,1,1,1) 
x1b = torch.clone(x1a)*0.5
x1c = torch.clone(x1a)*0.3
x1 = torch.cat((x1a,x1b,x1c), dim=1) 
```
to this:
```
x1a = torch.ones(20,1,1,1)
x1 = torch.cat((x1a,x1b,x1c), dim=1) 
```

**Comment by Hazem-Ahmed-Abdelraouf** [comment]
Hello Jeremy, the cell outputs have been restored and for the `clone` statements I believe keeping them will help with the understanding of the example test cases for learners looking into the documentation.

**Comment by jph00** [comment]
Thank you!
